v2/internal/app/app_dev.go

```diff
@@ -110,6 +116,9 @@ func CreateApp(appoptions *options.App) (*App, error) {
 		if loglevelFlag != nil {
 			loglevel = *loglevelFlag
 		}
+		if obfuscateFlag != nil && *obfuscateFlag {
+			obfuscate = fmt.Sprintf("%s", *obfuscateFlag)
```

Comment:
Shouldn't we format here with `%t`, since it's a bool?
```suggestion
			obfuscate = fmt.Sprintf("%t", *obfuscateFlag)
```